app/core/bookmarks.js

```diff
@@ -95,12 +95,12 @@ function(app, Backbone, EntriesManager) {
             table = "directus_" + table;
           }
           if(title && table) {
+            var that = this;
             app.router.openModal({type: 'confirm', text: 'Are you sure you wish to delete the snapshot: ' + title, callback: function() {
-              var user = app.users.getCurrentUser().get("id");
```

Comment:
@benhaynes do you know why this four lines were here,, I deleted because I could resolve the problem with the news lines.